tensorflow/lite/experimental/micro/micro_interpreter.cc

```diff
@@ -83,9 +83,67 @@ MicroInterpreter::MicroInterpreter(const Model* model,
   context_.impl_ = static_cast<void*>(this);
   context_.ReportError = ReportOpError;
   context_.recommended_num_threads = 1;
+
+  initialization_status_ = AllocateInputAndActTensors();
```

Comment:
I don't see the implementations of `AllocateInputAndActTensors()` or `AllocateTemporaryTensors`. Are they meant to be part of this PR, or has a future change leaked in here?

Reply:
They were originally in this PR but they appear to have been removed in the trunk since I forked this branch. The latest version of micro_interpreter.cc on Tensorflow master doesn't have these methods at all and has other changes too. 

If I remove the calls within the constructor from this PR, then this should be cleared up when resolving the merge conflicts. The latest version calls AllocateTensors() within the Invoke method instead.